pkg/services/postgresql/database_bind.go

```diff
@@ -0,0 +1,95 @@
+package postgresql
```

Comment:
Something else you may notice-- I renamed many files so that their names denote manager first and function second. Visually, this keeps things that are more related grouped together in the tree.

Reply:
👍